test/e2e/vsphere_utils.go

```diff
@@ -265,3 +267,74 @@ func getVSpherePodSpecWithClaim(claimName string, nodeSelectorKV map[string]stri
 	}
 	return pod
 }
+
+// func to get pod spec with given volume paths, node selector lables and container commands
+func getVSpherePodSpecWithVolumePaths(volumePaths []string, keyValuelabel map[string]string, commands []string) *v1.Pod {
+	var volumeMounts []v1.VolumeMount
+	var volumes []v1.Volume
+
+	for index, volumePath := range volumePaths {
+		var name = "volume" + strconv.Itoa(index+1)
```

Comment:
nit: I see something like this more often, though perhaps your method is more efficient?
`name = fmt.Sprintf("volume%v", index+1)`